externs/olx.js

```diff
@@ -7443,6 +7444,16 @@ olx.tilegrid.TileGridOptions.prototype.extent;
 
 
 /**
+ * Zoom levels to load from a tile source. This array defines a subset of the
+ * available zoom levels. When configured, only tiles for the specified zoom
+ * levels will be requested from a tile source.
+ * @type {Array.<number>|undefined}
+ * @api
+ */
+olx.tilegrid.TileGridOptions.prototype.loadZooms;
```

Comment:
I'm adding tests right now, then it should be clear. But a quick explanation: If you pass [8, 4, 2, 1, 0.5] as `resolutions`, then there will be 5 zoom levels. So a `loadZooms` configuration of [0, 3] would result in tiles for resolutions 8 and 1 to be loaded.